src/components/SignUp/index.scss

```diff
@@ -0,0 +1,99 @@
+@import "../../style/variables";
+@import "../../style/buttons";
+@import "../../style/headings";
+@import "../../style/bootstrapImports";
+
+.signUpSection {
+  /* All section style */
+  width: 100%;
+  margin-top: 4rem;
+  margin-bottom: 4rem;
+  background-color: $lightColor;
+
+  .signInImg {
+    background-color: $lightColor;
+    margin: auto 0;
+
+    img {
+      width: 100%;
+    }
+  }
+
+  h2 {
+    @include mainHeading;
+    font-size: 2.8rem;
+    color: $primaryColor;
+    margin-bottom: 4rem;
+  }
+
+  h4 {
+    font-size: 1.25rem;
+    color: $primaryColor;
+    margin-bottom: 2rem;
+
+    &::before {
+      display: inline-block;
+      border-top: 0.4rem solid #fe8f7c;
+      border-radius: 10px;
+      width: 56px;
+      height: 0px;
+      content: "";
+      margin: auto 0.5rem auto 0;
+    }
+  }
+
+  a {
+    color: $primaryColor;
+
+    &:hover {
+      text-decoration: none;
+      color: $warningColor;
+    }
+  }
+
+  hr.divider {
+    overflow: visible;
+    padding: 0;
+    margin-bottom: 10px;
+    border: none;
+    border-top: 1px solid #e0e0e0;
+    color: #6e6d7a;
+    text-align: center;
+
+    &::after {
+      content: "Or";
+      display: inline-block;
+      position: relative;
+      top: -0.8rem;
+      font: normal 14px/14px "Haas Grot Text R Web", "Helvetica Neue", Helvetica,
```

Comment:
I wonder why does this need a different font?

Reply:
I forgot to remove it